lib/tether/tether.go

```diff
@@ -212,6 +217,15 @@ func (t *tether) Start() error {
 		}
 		extraconfig.Encode(t.sink, t.config)
 
+		log.Info("Populating ext4 block device list now.")
+		deviceManager := fs.Ext4DeviceManager{}
```

Comment:
Very `ext` specific.  How do you intend to extend this to support NFS or NTFS?  We should hide these implementations in `BlockDevices`.